packages/common-all/src/types/index.ts

```diff
@@ -63,9 +63,19 @@ export type NotesCache = {
   notes: NotesCacheEntryMap;
 };
 export type NotesCacheEntryMap = { [key: string]: NotesCacheEntry };
-export type NotesCacheEntry = {
+export type NotesCacheEntry =
+  | {
+      hash: string;
+      data: Omit<NoteProps, "body">;
+      status?: "create" | "delete";
```

Comment:
Are we actually using this `status` prop anywhere? If not, can we remove it?

Reply:
I have a followup pr that will use it for backlinks caching. Mostly put it here so people can see the schema changes before I do the implementations. Hope you don't mind if I leave it here

Reply:
Actually I'm going to remove. Turns out my next pr will do things a little differently anyways